browser_history/generic.py

```diff
@@ -40,12 +40,15 @@ class Browser(abc.ABC):
       This path should be one level above the profile directories if they
       exist. At least one of the following must be set:
       :py:class:`windows_path`, :py:class:`mac_path`, :py:class:`linux_path`
-    * :py:class:`profile_support`
-    * :py:class:`profile_dir_prefixes`
     * :py:class:`history_file`
-    * :py:class:`bookmarks_file`
     * :py:class:`history_SQL`
+
+    These following class variable can optionally be set:
+
+    * :py:class:`bookmarks_file`
     * :py:class:`bookmarks_parser`
+    * :py:class:`profile_dir_prefixes`
+    * :py:class:`profile_support`
```

Comment:
Just a very small thing. IMO, `profile_support` should come before `profile_dir_prefixes`